helmfile/charts/user-rbac/templates/clusterrolebindings/extra-user-view.yaml

```diff
@@ -0,0 +1,9 @@
+# Created, but not updated by compliantkubernetes-apps
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: extra-user-view
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: user-view
```

Comment:
Will this not break if we, at some point, would need to change something here?
I would rather like to see that we have something that looks like this and then is just deployed with a `kubectl apply -f ...` maybe in `scripts/deploy-wc.sh`. Then this would be unaffected if we were to change something in this (e.g. add a label etc.)

Reply:
Thanks !! I have added it now ,